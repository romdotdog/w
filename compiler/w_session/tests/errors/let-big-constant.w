fn main() {
    let a: i32 = 2147483648;
}