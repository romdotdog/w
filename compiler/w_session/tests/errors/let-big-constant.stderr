tests/errors/let-big-constant.w:2:12: error: type mismatch
    let a: i32 = 2147483648;
           ^^^
